components/ActionSheet/src/MDCActionSheetController.m

```diff
@@ -175,7 +179,14 @@ - (void)viewWillLayoutSubviews {
   }
   CGSize size = [self.header sizeThatFits:CGRectStandardize(self.view.bounds).size];
   self.header.frame = CGRectMake(0, 0, self.view.bounds.size.width, size.height);
-  UIEdgeInsets insets = UIEdgeInsetsMake(self.header.frame.size.height, 0, 0, 0);
+  if (self.title != nil || self.message != nil) {
+    [self.view addSubview:self.headerDividerView];
```

Comment:
`-[UIView addSubview:]` requires a `nonnull` argument and it looks like `headerDividerView` can be `nil` at this point if the divider color is `nil`.

Reply:
The headerDividerView is no nonnull.